optional_struct_macro/src/opt_struct.rs

```diff
@@ -519,6 +521,16 @@ impl Parse for ParsedMacroParameters {
             return Ok(out);
         };
 
+        if input.parse::<Token![,]>().is_err() {
+            return Ok(out);
+        };
+
+        if let Ok(wrapping) = syn::LitBool::parse(input) {
```

Comment:
`wrapping` should be renamed to something else, e.g. `skip_struct`

Reply:
Yes, the original attribute was named "wrapping", which was changed to the at least slightly better name "skip_struct".    Depending on the compiler to tell me where I need to change the name is useful, but misses places like this.